plugins/pulp_rpm/plugins/db/models.py

```diff
@@ -62,6 +64,28 @@ def clone(self):
         son_data.pop('_id')
         return self.__class__(**son_data)
 
+    def to_id_dict(self):
```

Comment:
We're going to make a post-merge release note to merge these two functions together later.